sequins.go

```diff
@@ -150,7 +168,14 @@ func (s *sequins) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	key := strings.TrimPrefix(r.URL.Path, "/")
-	res, err := s.index.Get(key)
+
+	currentIndex := s.indexMonitor.AcquireRef()
+	if currentIndex == nil {
+		log.Fatal("Tried to make HTTP request before initializing the index") 
+	}
```

Comment:
Ah, actually, this is not possible at all (unless you're me, and didn't set the test up right at first). I'll get rid of this line, since it's pretty useless.